dockerfiles/build.include

```diff
@@ -137,14 +168,14 @@ build_image() {
     -e "s;\${BUILD_PREFIX};${PREFIX};" \
     -e "s;\${BUILD_TAG};${TAG};" \
     > ${DIR}/.Dockerfile
-  cd "${DIR}" && docker build -f ${DIR}/.Dockerfile -t ${IMAGE_NAME} ${BUILD_ARGS} .
+  cd "${DIR}" && "${BUILDER}" build -f ${DIR}/.Dockerfile -t ${IMAGE_NAME} ${BUILD_ARGS} .
```

Comment:
If you're adding support for `buildah` then you need to add the `BUILD_COMMAND` option here, and make sure it's `buildah bud` or `podman build` or `docker build`.

Reply:
done